src/nodeOperation.js

```diff
@@ -23,19 +23,32 @@ let $d = document
 export let updateNodeStyle = function (object) {
   if (!object.style) return
   let nodeEle = findEle(object.id, this)
+  let origin = {
+    color: nodeEle.style.color && rgbHex(nodeEle.style.color),
```

Comment:
🤔 I think `rgbHex` is unnecessary

Reply:
rgbHex was added so that the input and output was the same format. When user listens fo the 'editStyles' operation, they'll get the new style (in hex) as well as the old style (in hex). Otherwise, they'll get new as hex and old as rgb

Reply:
But what if they just need rgb 😂
I mean user can transform data form by themselves.

Reply:
seems kind to the dev who needs to do the comparison. why have different format for input and output?

Reply:
😂sorry, i don't realize that the browser transform hex to rgb automatically